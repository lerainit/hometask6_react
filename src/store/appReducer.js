import { combineReducers } from "redux";
import cardIndexReducer from "./cardId/reducer";
import modalReducer from './modal/reducer'
import modalTextReducer from "./modalText/reducer";
import CartReducer from "./addCards/reducer";
import addFavoritesReducer from "./addFavorites/reducer";
import productsReducer from "./products/reducer";


const appReducer = combineReducers({
   modal:modalReducer,
   cardIndex:cardIndexReducer,
   modalText:modalTextReducer,
   Cart:CartReducer,
   addFavorites:addFavoritesReducer,
   products:productsReducer

})
export default appReducer