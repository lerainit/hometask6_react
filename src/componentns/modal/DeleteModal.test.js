
import { render,} from "@testing-library/react";
import DeleteModal from './deletemodal'
import { Provider } from "react-redux";
import store from '../../store/'

const Component =() =>{
    return (
    <Provider store ={store}>
        <DeleteModal/>
    </Provider>
)}
describe('Modal renders',()=>{

test('should Modal match snapshot',()=>{
const {asFragment } = render(<Component/>)
expect(asFragment()).toMatchSnapshot()

})

})
