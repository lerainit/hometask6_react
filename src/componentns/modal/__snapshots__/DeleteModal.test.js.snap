// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Modal renders should Modal match snapshot 1`] = `
<DocumentFragment>
  <div
    class="modal"
  >
    <div
      class="outer_container"
    />
    <div
      class="modal_main_container"
    >
      <button
        class="close_btn"
      >
        X
      </button>
      <p>
        Are  you sure you want to add this product to cart?
      </p>
      <button
        class="modal_btn"
      >
        delete from Cart
      </button>
    </div>
  </div>
</DocumentFragment>
`;
